admin/app/clusters/[clusterId]/tasks/page.tsx

```diff
@@ -0,0 +1,201 @@
+"use client";
+
+import { client } from "@/client/client";
+import { DataTable } from "@/components/ui/DataTable";
+import { useAuth } from "@clerk/nextjs";
+import { ScrollArea } from "@radix-ui/react-scroll-area";
+import { formatRelative } from "date-fns";
+import { useEffect, useState } from "react";
+import { Dot } from "react-animated-dots";
+import toast from "react-hot-toast";
+import { functionStatusToCircle } from "../helpers";
+import Link from "next/link";
+import { Button } from "@/components/ui/button";
+
+export default function Page({ params }: { params: { clusterId: string } }) {
+  const { getToken, isLoaded, isSignedIn } = useAuth();
+
+  const [data, setData] = useState<{
+    loading: boolean;
+    taskId: string | null;
+    result: string | null;
+    jobs: {
+      id: string;
+      createdAt: Date;
+      targetFn: string;
+      status: string;
+      functionExecutionTime: number | null;
+    }[];
+  }>({
+    loading: false,
+    taskId: null,
+    result: null,
+    jobs: [],
+  });
+
+  useEffect(() => {
+    const fetchData = async () => {
+      if (!data.taskId) {
+        return;
+      }
+
+      const clusterResult = await client.getClusterDetailsForUser({
```

Comment:
This is really just to demo as it will only show _recent_ job executions and files them on the FE for now.